packages/shared/src/lib/feed.ts

```diff
@@ -210,3 +210,8 @@ export const getFeedName = (
 
   return page.replace(/^\/+/, '') as SharedFeedPage;
 };
+
+export type FeedAdTemplate = {
+  adStart: number;
+  adRepeat?: number;
+};
```

Comment:
I can make this Record<screen_size, FeedAdTemplate> if we want to customize per tablet/mobile/laptop.

Reply:
Yeah think we need to right, to control existing behaviour?

Reply:
Current `adSpot`, behaviour is static and not per screen size https://github.com/dailydotdev/apps/blob/ca508b46cb94ae935f7b22994cdebc255da51839/packages/shared/src/components/Feed.tsx#L173

Reply:
For list and squad we use always `2` so I left that as is since the previous implementation also did not touch it.

Reply:
Yep, will adjust!

Reply:
Added in cb5d1e0b587d3119715644b3712ae1ef7b4b4ddc